compiler/ras/Tree.cpp

```diff
@@ -997,6 +986,26 @@ void TR_PrettyPrinterString::append(const char* format, ...)
 
    }
 
+void TR_PrettyPrinterString::appends(char const *str)
+   {
+   size_t const strLen0 = strlen(str) + 1; // include terminating '\0'
+   size_t const bufLenBytes = 2000-len;
```

Comment:
The 2000 should be made into a const static in the `TR_PrettyPrinterString` class.

Reply:
Fixed in force push.